db/seeds.rb

```diff
@@ -51,7 +51,7 @@
   },
   {
     application: "LTI Starter App",
-    tenant: "starter-app",
+    tenant: Rails.application.secrets.default_lti_key,
```

Comment:
In the apartment.rb it's selecting the tenant names by the lti keys -- config.tenant_names = lambda { ApplicationInstance.pluck(:lti_key) }

So the tenant name and the lti_key ought to be the same, so I was just adding changing it to be the same as whatever the lti_key was being set to. The admin one above are both strings as the lti_key and tenant. But I can change it to a string of what the default_lti_key is.